src/ASM/LR/ASMu2D.C

```diff
@@ -1270,7 +1271,8 @@ bool ASMu2D::integrate (Integrand& integrand,
 #endif
 
         // Cartesian coordinates of current integration point
-        X = Xnod * fe.N;
+        X.assign(Xnod * fe.N);
+        X.u = elmPts[ip].data();
```

Comment:
Don't think you need to switch to using the assign-method here (and in lines 1220 and 1115) since I here use `X.u = (pointer)` instead of going via a double[3] temporary.

Reply:
done, but 1115 seems a necessity to me.

Reply:
yes, that was in another integrate method, my mistake